src/System.Reflection.Emit.ILGeneration/tests/ILGenerator/Emit4Tests.cs

```diff
@@ -0,0 +1,139 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Linq;
+using System.Runtime.InteropServices;
+using Xunit;
+
+namespace System.Reflection.Emit.Tests
+{
+    public class ILGeneratorEmit4
+    {
+        [Fact]
+        public void TestEmitCalliStdCall()
```

Comment:
I think that the key test coverage difference between the two flavors is int vs. string.

I would call this method `TestEmitCalliBlittable` and the other method `TestEmitCalliNonBlittable`.